### ISSUE TITLE
1.12 crash when editing null

### ISSUE THREAD

**Issue by redgoblin88** [FTBTeam/FTB-Quests#530]
forge: 14.23.5.2854
ftb questing: 1202.9.0.15

i made one sub-quest depend on another quest, but i accidently clicked the "+" icon twice, so i essentially made one quest requirement, and also a second requirement pointing to nothing
but it seems when i closed the GUI for making the requirements the null value remained. upon going back into the "quest dependencies" it seems the mod did not like trying to load up a null pointer

in other words: questA depends on quest B but i accidently added a secondry A requiring NULL but that value failed to remove itself

crashlog: https://pastebin.com/Ybk1QFTb

**Comment by MaxNeedsSnacks** [comment]
You should be able to edit that quest manually by going to the relevant .snbt file, but as you may know 1.12 is unsupported anyways. If this issue still exists on 1.16, we'll probably try to fix it there (and backport if possible)
